code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/OpenLDAP/OpenLDAPIdentityProvider.java

```diff
@@ -70,7 +70,7 @@
                 }
                 Set<Identity> identities = getIdentities(dn);
                 Token token = getTokenUtils().createToken(identities, null);
-                authToken = authTokenDao.createToken(token.getJwt(), getConstantsConfig().getConfig(), account.getId());
+                authToken = authTokenDao.createToken(token.getJwt(), getConstantsConfig().getConfig(), account.getId(), account.getId());
             }
```

Comment:
@prachidamle Confirmed that this code is hit only when creating a token by using api keys, not through browser session